transcription/transcription.go

```diff
@@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
+func ConvertAudioIntoRequiredFormat(fn string) error {
```

Comment:
Tried resolving this problem for some time, not sure how to: <img width="407" alt="screen shot 2016-03-12 at 10 54 40 am" src="https://cloud.githubusercontent.com/assets/4250521/13723903/32cc0568-e841-11e5-9edc-5b5b5b712d4c.png">